neurodsp/burst.py

```diff
@@ -7,7 +7,7 @@
 import numpy as np
 
 
-def detect_bursts(Fs, x, f_range, algorithm, thresh, magnitudetype='amplitude',
+def detect_bursts(x, Fs, f_range, algorithm, thresh, magnitudetype='amplitude',
```

Comment:
can you also change the ordering of these parameters in the docstring

Reply:
Done